Makefile

```diff
@@ -87,7 +87,7 @@ build:
 	make stop
 
 clean: deps
-	rm -rf $(GOPATH)/pkg/*
```

Comment:
This is will not work on mac . You might want to check for the os before using linux_amd64 vs darwin_amd64 ?